src/cli.rs

```diff
@@ -22,6 +27,19 @@ pub struct L1FetcherOptions {
     pub disable_polling: bool,
 }
 
+impl From<L1FetcherOptions> for FetcherOptions {
```

Comment:
Nice! We should mention why we implement `From` though. I think a common question, if someone looks at the code base, would be: "Why not just use `cli::L1FetcherOptions`" in `L1Fetcher` directly?".

Reply:
I wouldn't dream of creating a dependency like that, let alone ask the question... :-) But sure, I can add a comment...